windows/server_stats.ps1

```diff
@@ -1,32 +1,57 @@
 # Note, all code to be executed on the remote server needs to belong
 # in the ServerStats code block
 $ServerStats = {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory)]
+        [string]
+        $ComputerName,
+
+        [Parameter()]
+        [pscredential]
+        $Credential,
+
+        [Parameter()]
+        [switch]
+        $ProcessStats
+    )
+    $getWmiObjectParams = @{
+        ComputerName  = $ComputerName
+        Namespace     = "ROOT\cimv2"
+        Impersonation = 3 # Impersonate. Allows objects to use the credentials of the caller.
+    }
+    $remote = $ComputerName -notin ".", "localhost", ([System.Environment]::MachineName)
```

Comment:
It looks like we use this `$remote` to determine a lot of logic. I feel it gets kind of buried in between the lines. 
Can we add a comment to add some visibility about the purpose and need to have this value.